ui/run.sh

```diff
@@ -118,10 +118,11 @@ case "$(basename ${0})" in
         run ecsdeploy check || exit $?
         run ecsdeploy bootstrap || exit $?
         run ecsdeploy reboot || exit $?
-        sleep 10
+        sleep 30
```

Comment:
Use the correct API calls to determine cluster readiness, not sleep.

Reply:
This sleep was already their , it just increased to 30 seconds from 10 seconds.